lib/compress/fse_compress.c

```diff
@@ -25,6 +23,8 @@
 #define FSE_STATIC_LINKING_ONLY
 #include "../common/fse.h"
 #include "../common/error_private.h"
+#define ZSTD_DEPS_NEED_MALLOC
+#include "../common/zstd_deps.h"  /* malloc, free, memcpy, memset */
```

Comment:
minor : with all these little changes (not just this one),
we have made it more difficult to synchronize `fse` between its own library and the `zstd` repository.

It's now more firmly integrated as a specific `zstd` unit,
making it a bit less generic, hence more difficult to re-use elsewhere

Maybe something to think about, though likely a topic for a different PR.